tests/TestCase/Console/ConsoleIoTest.php

```diff
@@ -257,23 +222,17 @@ public function testQuietOutput(): void
     /**
      * testErr method
      */
-    public function testErr(): void
+    #[TestWith(['Just a test'])]
+    #[TestWith([['Just', 'a', 'test']])]
+    #[TestWith([['Just', 'a', 'test'], 2])]
+    #[TestWith([''])]
```

Comment:
This is an interesting pattern. It is a bit more readable than a dataprovider method.

Reply:
Yes, not having to scroll down/up to the provider method to see the values passed is nice.